apple/swift-docc-render
# It allows set a list of available locales per page coming from the metadata object in the JSON file

Bug/issue #109166355, if applicable: 

## Summary

It allows set a list of available locales per page coming from the metadata object in the JSON file

## Dependencies

NA

## Testing

[SlothCreatori18n.doccarchive.zip](https://github.com/apple/swift-docc-render/files/11628329/SlothCreatori18n.doccarchive.zip)

Steps:
1. Add the following to `theme-settings.json` file at public folder
```json
{
  "features": {
    "docs": {
      "i18n": {
        "enable": true
      }
    }
  }
}
```
2. Run attached doccarchive
3. Go to http://localhost:8080/documentation/slothcreator
4. Assert that you can see on the bottom of the page the language selector with 2 languages on it
5. Go to any other page
6. Assert that you cannot see the language selector anymore

## Checklist

Make sure you check off the following items. If they cannot be completed, provide a reason.
